lib/openhab/dsl/monkey_patch/items/metadata.rb

```diff
@@ -134,6 +134,20 @@ def []=(namespace, value)
               end
             end
 
+            #
+            # Implements Hash#dig-like functionaity to metadata
+            #
+            # @param [String, Symbol] key The first key
+            # @param [Array<String, Symbol>] keys More keys to dig deeper
+            #
+            # @return [OpenHAB::DSL::MonkeyPatch::Items::MetadataItem], or nil if the namespace doesn't exist
+            #
+            def dig(key, *keys)
+              return self[key.to_s] if keys.empty?
+
+              key?(key) ? self[key.to_s].dig(*keys) : nil
```

Comment:
Stylistic preference, I suppose? Your version makes it more obvious that it would return nil if key is not defined. Below is shorter. Which one is more common in Ruby?

```suggestion
              self[key.to_s].dig(*keys) if key?(key)
```

Reply:
I think your version looks more ruby actually :)